### PR TITLE
Fix test not use test flags

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
old code not use test.flags, now use it

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by derekwaynecarr** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by zjj2wry** [comment]
cc @kubernetes/sig-cli-pr-reviews can someone give this a lgtm?

**Comment by shiywang** [comment]
@mengqiy mind give an approval ? I think tony is quit busy recently

File: pkg/kubectl/cmd/create_quota_test.go [hunk 1]
```diff
@@ -47,30 +47,30 @@ func TestCreateQuota(t *testing.T) {
 	tf.Namespace = "test"
 
 	tests := map[string]struct {
-		flags          map[string]string
+		flags          []string
 		expectedOutput string
 	}{
 		"single resource": {
-			flags:          map[string]string{"hard": "cpu=1", "output": "name"},
```
Comment (sttts) [inline pkg/kubectl/cmd/create_quota_test.go]: `output` not needed anymore?

Comment (shiywang) [inline pkg/kubectl/cmd/create_quota_test.go]: output already set in line 74 `cmd.Flags().Set("output", "name")` I guess

Comment (sttts) [inline pkg/kubectl/cmd/create_quota_test.go]: Right, so it used to be set twice.

**Comment by sttts** [comment]
lgtm. @adohe approved?

**Comment by zjj2wry** [comment]
@mengqiy @fabianofranz mind give an approval ? I think tony is quit busy recently

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-unit test this

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by zjj2wry** [comment]
@grodrigues3 @deads2k mind for approve?

**Comment by zjj2wry** [comment]
@fabianofranz mind for approve? the first approver not online recently. sorry to about ping you many times. think you very much

**Pull request merged**